Library/Formula/wxmac.rb

```diff
@@ -67,8 +68,13 @@ def install
       "--disable-monolithic"
     ]
 
-    system "./configure", *args
+    system "./configure", "--enable-shared", *args
```

Comment:
Thanks for the detailed explanation :bow:. Perhaps instead of redoing the whole `make install` process again you could just do `make` and then `lib.install Dir["*.a"]` or something?